### DISCUSSION TITLE
Get-D365LcsDatabaseBackups - What do I do wrong?

### DISCUSSION THREAD

**Discussion by Myhlendorph** [d365collaborative/d365fo.tools#526] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/37613912/112321774-a24cb180-8cb0-11eb-803a-f7de06e3233e.png)

Hi
I have just deployed a new LCS D365FO Developer VM and installed the newest d365fo.tools + azcopy
I follow the guide for the Get-D365LcsDatabaseBackups and I get the attached error.
The account that I use is LCS owner and owner for the App Registration.
What do I do wrong?
Thank you in advance :-)

**Comment by Splaxi** [comment, marked as answer]
Hi @myhlendorph 

Did you try the guide we have on the wiki?
This is for the bacpac files, but it uses the same mechanics: https://github.com/d365collaborative/d365fo.tools/wiki/How-To-Download-Latest-Bacpac-From-Lcs

This blog shows some of the other cmdlets against LCS, but shows what you need to have in place, to work with the LCS API: https://msdax.wordpress.com/2021/01/11/d365fo-tool-automating-start-and-stop-environments/

**Reply by Myhlendorph** [reply]
Hi @Splaxi
Went through the guide step by guide and what did the the trick was to change the LcsApiUri from [https://lcsapi.eu.lcs.dynamics.com] to [https://lcsapi.lcs.dynamics.com] and now it's working beautifully.
Thank you and thank you for the d365fo.tools that helps me a lot working with the D365 infrastructure.
Have a nice easter :-)
